ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -56,4 +62,9 @@ private static String getServerIpAddress(int port) {
 
     return String.format(Locale.US, "%s:%d", ipAddress, port);
   }
+
+  public static boolean isRunningOnTV(ReactContext reactContext) {
```

Comment:
This code does not belong here. This class provides methods that are used by the code loader (most typically in debug mode). You can just put this code directly in `AndroidInfoModule` as it is not being used anywhere outside of that module (as opposed to other methods from this class).

Also I'd change the naming strategy here. With "running" it looks weird as we don't use "isRunningOnAndroid", just "platform == 'android'". Perhaps we could find something similar to "interfaceIdiom" exported on iOS. It looks like maybe "uiMode" would be a good choice as it could be TV, NORMAL, CAR etc (https://developer.android.com/reference/android/content/res/Configuration.html#uiMode) much like interfaceIdiom on iOS